common/tools/dev-tool/src/commands/run/bundle.ts

```diff
@@ -70,9 +70,12 @@ export default leafCommand(commandInfo, async (options) => {
 
     try {
       const bundle = await rollup.rollup(baseConfig);
-
+      const cjsOutput = info.packageJson.main;
```

Comment:
nitpick: Is it just me or the var name feels weird?

Reply:
yeah, bundleOutput may be better, but probably not worth restarting CI checks